Makefile

```diff
@@ -1,10 +1,11 @@
 # ################################################################
-# Copyright (c) 2015-present, Yann Collet, Facebook, Inc.
+# Copyright (c) 2015-2020, Yann Collet, Facebook, Inc.
 # All rights reserved.
 #
 # This source code is licensed under both the BSD-style license (found in the
 # LICENSE file in the root directory of this source tree) and the GPLv2 (found
 # in the COPYING file in the root directory of this source tree).
+# You may select, at your option, one of the above-listed licenses.
```

Comment:
'at your *discretion*' sounds more natural to me than 'at your *option*'

Reply:
I agree, but to change it we would have to talk to our lawyers, and it isn't worth the hassle unless the wording is causing problems for someone.